### PR TITLE
fix(bare-metal-cloud): fix cloud connect issues

### PR DESCRIPTION
**Pull request by varun257**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-22072 Fix #DTRSD-22065
| License          | BSD 3-Clause

## Description

fix translation issues
fix service key regenerate disable button

### CODE REVIEW

File: packages/manager/modules/cloud-connect/src/cloud-connect.class.js [hunk 1]
```diff
@@ -164,11 +164,16 @@ export default class CloudConnect {
...
+    set(this, 'validServiceKey', false);
+    map(serviceKeys, ({ status }) => {
+      if (
+        status === STATUS.ACTIVE ||
+        status === STATUS.DOING ||
+        status === STATUS.TO_CHECK
+      ) {
+        set(this, 'validServiceKey', true);
+      }
+    });
```
Comment (cbourgois) [inline packages/manager/modules/cloud-connect/src/cloud-connect.class.js:176]: Please avoid to use `lodash`
```suggestion
    this.validServiceKey = serviceKeys.some(({status}) => [STATUS.ACTIVE, STATUS.DOING, STATUS.TO_CHECK].includes(status));
    this.serviceKeys = serviceKeys;
```

**Review by cbourgois — CHANGES_REQUESTED** [review]

File: packages/manager/modules/cloud-connect/src/cloud-connect.class.js [hunk 2]
```diff
@@ -164,11 +164,10 @@ export default class CloudConnect {
   }
 
   setServiceKeys(serviceKeys) {
-    if (find(serviceKeys, { status: STATUS.ACTIVE })) {
-      set(this, 'validServiceKey', true);
-    } else {
-      set(this, 'validServiceKey', false);
-    }
+    set(this, 'validServiceKey', false);
```
Comment (marie-j) [inline packages/manager/modules/cloud-connect/src/cloud-connect.class.js:167]:
```suggestion

```

**Review by marie-j — COMMENTED** [review]
The scope of the commit `bare-metal-cloud` is not following any module/app architecture. As all the changes are located in the occ module maybe we can split into two commits
`fix: update wording` and `fix: update service key check`

**Review by JeremyDec — DISMISSED** [review]

**Review by cbourgois — APPROVED** [review]

**Comment by ovh-cds** [comment]
CDS Report translator-push#13201.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#13201.1 ✘  
* 
  * push ✘

**Review by antleblanc — DISMISSED** [review]

**Comment by antleblanc** [comment]
Rebasing on `develop` branch.

**Comment by antleblanc** [comment]
Rebased successfully on `develop` branch.

**Review by antleblanc — APPROVED** [review]

**Pull request merged**